components/aws/sagemaker/tests/integration_tests/sample_pipeline_tests/test_sample_train_pipeline.py

```diff
@@ -0,0 +1,77 @@
+import pytest
```

Comment:
merge this with training component test ?,

Reply:
done